src/site/content/en/blog/earth-webassembly/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "How we’re bringing Google Earth to the Web"
+subhead: |
+  Improving cross-browser access to Google Earth with WebAssembly.
+date: 2019-06-20
+authors:
+  - jormears
+hero: hero.jpg
+alt: A globe with code overlaid
+description: |
+  Improving cross-browser access to Google Earth with WebAssembly
+tags:
+  - post
+  - wasm
+  - WebAssembly
+  - Earth
+  - Web App
+---
+
+In an ideal world, every application that developers build, regardless of technology, would be easily made available in the browser. But there are barriers to bringing projects to the web, depending on the technology they were built with and how well that technology is supported by the various browser vendors. [WebAssembly](https://webassembly.org/) (Wasm) is a compile target standardized by the [W3C](https://www.w3.org/) that helps us solve this problem by allowing us to run codebases from languages other than JavaScript on the web.
+
+
+We’ve done just that with Google Earth, available today in [preview beta](https://g.co/earth/beta) on WebAssembly. Keep in mind that this is still a beta of Google Earth and may not be as smooth as you’re used to (try out regular [Earth for web](https://earth.google.com/web/)). You can experiment with this beta in Chrome and other Chromium-based browsers, including Edge (Canary version) and Opera, as well as Firefox. Consider this beta your inspiration if you too are looking for better cross-browser support for your native applications.
+
+
+## Why we chose WebAssembly for Google Earth
+
+We originally wrote most of Google Earth in C++ because it was initially a native application, intended for desktop install. Then we were able to port it to Android and iOS as smartphones took hold and retain most of our C++ codebase using [NDK](https://developer.android.com/ndk) and [Objective-C++](https://www.wikipedia.org/wiki/Objective-C#Objective-C++). 
```

Comment:
Small suggestion to smooth over first sentence: "because it was a native application intended for desktop install'. Removed 'initially' and the unnecessary comma.

Reply:
fixed